### DISCUSSION TITLE
delete the "update-target"

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-tycho/tycho#1775] [General]
Tycho extras contains a mojo called "update-target" that describes as

>  Quick&dirty way to update .target file to use latest versions of IUs available from specified metadata repositories.

There is only one invoker test and the mojo itself seems only doing some specific thing for IU locations what could be archived by the target editor in PDE as well. As it uses outdated API I like to delete the mojo if no one has a strong use-case for this and likes to update it.

**Comment by HannesWell** [comment]
No objections from me.
If one wants the latest version of a IU `0.0.0` should be used.
